src/test/ptrace_syscall.c

```diff
@@ -16,7 +16,9 @@ extern char syscall_addr __attribute__ ((visibility ("hidden")));
 int main(void) {
   pid_t child;
   int status;
+#ifdef PTRACE_GET_SYSCALL_INFO
```

Comment:
Just move the declaration into the ifdef below rather than adding another one? We're not imposing C89 in the tests ;)

Reply:
I've thought about this, but as we have that multiple times, I've left that above.

Reply:
BTW @Keno - if you still search for an Altra: Oracle Cloud free tier offers 4 core / 24 GB RAM (you _may_ need to set it down later to 1 core /  6 GB after 29 days to not loose the machine; but that is quite some time to use that really fast environment to compile your setup [in my case: recent binutils, gcc and gdb + full system update).

I didn't get `rr` to work at all there so I'm backing out, leaving the field to the pro's.